dev/io.openliberty.jakarta.validation.3.1_fat_tck/fat/src/io/openliberty/jakarta/validation/tck/FATSuite.java

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright (c) 2022, 2024 IBM Corporation and others.
```

Comment:
```suggestion
 * Copyright (c) 2024 IBM Corporation and others.
```
This file is new for 2024